### PR TITLE
Update Private Cloud Tekton instructions.

### PR DESCRIPTION
**Pull request by walkline**
Update supported Tekton version.
Update instructions to download pipeline in airgapped section.

This PR can be merged immediately.

### CODE REVIEW

**Review by Nidhi251289 — COMMENTED** [review]
Looks good to me

**Review by Nidhi251289 — APPROVED** [review]

**Comment by MarkvanMents** [comment]
Hi Kasia (@katarzyna-koltun-mx )
Can you have a look at this Private Cloud PR when you have a moment.
Thanks.

File: content/en/docs/developerportal/deploy/private-cloud/private-cloud-tekton/_index.md [hunk 1]
```diff
@@ -114,17 +114,20 @@ If Tekton is already installed in your namespace, you can skip to [Pipeline Inst
 
 ### 5.1 Installing on Connected Kubernetes
 
-To install Tekton with Tekton Triggers you need to apply 3 yaml manifests:
+To install Tekton with Tekton Triggers you need to apply 3 yaml manifests (main objects would be installed in `tekton-pipelines` namespace):
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/developerportal/deploy/private-cloud/private-cloud-tekton/_index.md]: @walkline Before I merge the PR, can you let me know if I understand this part correctly? Can I rephrase it as, "To install Tekton with Tekton Triggers, apply the following yaml manifests:

[codeblock]

Note: The manifests are installed in the tekton-pipelines namespace."

Is my understanding correct?

Comment (walkline) [reply content/en/docs/developerportal/deploy/private-cloud/private-cloud-tekton/_index.md]: `The manifests are installed ` some objects in the manifest are global, so it's not completely true. 
Maybe we can rephrase it to "The manifests are targeting ..."

Comment (katarzyna-koltun-mx) [reply content/en/docs/developerportal/deploy/private-cloud/private-cloud-tekton/_index.md]: Done.

File: content/en/docs/developerportal/deploy/private-cloud/private-cloud-tekton/private-cloud-tekton-airgap.md [hunk 2]
```diff
@@ -179,7 +179,7 @@ For Tekton Triggers on OpenShift you need to update the deployment objects to ma
 Before you install the Mendix pipelines, which contain all Tekton-related objects, you need to do the following:
 
 1. Install [helm](https://helm.sh).
-2. Create a folder containing helm charts for configuring the Mendix Tekton pipelines – you can download these from [Mendix for Private Cloud Standalone Tekton Pipelines](https://cdn.mendix.com/mendix-for-private-cloud/tekton-pipelines/standalone-cicd/standalone-cicd-v1.0.0.zip).
+2. Create a folder containing helm charts for configuring the Mendix Tekton pipelines – you can get these by making a request to your CSM, who can arrange for access to them.
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/developerportal/deploy/private-cloud/private-cloud-tekton/private-cloud-tekton-airgap.md]: @walkline One last question - does "you can get these" refer to helm charts or to Mendix Tekton pipelines? I'd like to rephrase this as, "To  get access to the [helm charts/Mendix Tekton pipelines], contact your CSM".

Comment (walkline) [reply content/en/docs/developerportal/deploy/private-cloud/private-cloud-tekton/private-cloud-tekton-airgap.md]: I think both would be correct. Mendix Tekton pipelines distributed as a helm charts. 
But I like more `helm charts` in this context.

Comment (katarzyna-koltun-mx) [reply content/en/docs/developerportal/deploy/private-cloud/private-cloud-tekton/private-cloud-tekton-airgap.md]: Done.

**Comment by katarzyna-koltun-mx** [comment]
@walkline I'm merging the PR. The updates should be available in production tomorrow.
